### DISCUSSION TITLE
Provide a way to send coffee

### DISCUSSION THREAD

**Discussion by jo-so-nx** [priv-kweihmann/oelint-adv#548] [Q&A] [answered]
You've done so much valuable work for our project in the last days, we would like to send you a little *Thank-you*. Do you have any kind of donation program?

**Comment by priv-kweihmann** [comment, marked as answer]
Thanks for the compliment, for tax reasons (German tax system is horrible), I abandoned the idea of a donation program, *but* if we meet in person I'm always in a for beer :-).
Usually you can catch me at least once a year on a Linux Foundation event
